src/tests/account/test_account.cairo

```diff
@@ -1,33 +1,22 @@
-use openzeppelin::account::Account::OwnerAdded;
-use openzeppelin::account::Account::OwnerRemoved;
-use openzeppelin::account::Account::PublicKeyCamelImpl;
-use openzeppelin::account::Account::PublicKeyImpl;
+use openzeppelin::account::Account::{InternalTrait, OwnerAdded, OwnerRemoved, SRC6CamelOnlyImpl};
```

Comment:
Should we group the events together as a standalone import like in https://github.com/OpenZeppelin/cairo-contracts/pull/794#discussion_r1366067973 ?